### DISCUSSION TITLE
iiab in a docker image

### DISCUSSION THREAD

**Discussion by WarHawk8080** [iiab/iiab#3486] [Ideas]
Why not create a docker image...then have all the persistent information point to a folder on the SD/SSD/HDD/USB 

Then using something like Portainer to be able to control/restart/configure the instance with ease

This way it can be installed/ran independent of the OS...or the architecture

**Comment by holta** [comment]
Question for @warhawk8080: have you also considered earlier/similar thoughts below?

- https://github.com/iiab/iiab/discussions/2852
- https://wiki.iiab.io/go/FAQ#What_OS_should_I_use%3F

**Comment by suoko** [comment]
@WarHawk8080 did you try to run the install script in an ubuntu 22.04 docker container?
Apart from the WiFi hotspot everything should juat work. You might need just to expose the ports from the container to the host

**Reply by jvonau** [reply]
One can suppress the installation of the network components by adding to /etc/iiab/local_vars.yml
```
network_install: False
network_enabled: False
```
These are developer based options and are not part of the stock local_vars.yml see https://github.com/iiab/iiab/wiki/IIAB-Networking#8-can-i-opt-out-of-networking
